src/fiber/channel.rs

```diff
@@ -178,7 +178,7 @@ pub struct AddTlcCommand {
     /// Save it for outbound (offered) TLC to backward errors.
     /// Use all zeros when no shared secrets are available.
     pub shared_secret: [u8; 32],
-    pub previous_tlc: Option<(Hash256, u64)>,
+    pub previous_tlc: Option<(Hash256, u64, u128)>,
 }
```

Comment:
how to do migration for this change?

Reply:
we don't save the `AddTlcCommand` in storage, I don't think we need migration for it.